### ISSUE TITLE
Multiple architecture support

### ISSUE THREAD

**Issue by MaddieM4** [MaddieM4/mm4cc-pkg#1]
This is good to do as early as possible, because if we try to retrofit it later, we'll deal with many problems at once instead of confronting them incrementally.

**Comment by MaddieM4** [comment]
There needs to be some single list of architectures supported, which gets sourced multiple places. I'm thinking `.env` is good, as it'll be nicely compatible for any uses within `docker-compose.yml`.

**Comment by MaddieM4** [comment]
I did get this to work eventually, but I don't like the imprecision with which I threw stuff at the wall until it worked. Still, here's a compressed summary of what I did, which will hopefully be possible to boil down in the future to what's actually necessary and helpful.

```bash
docker buildx create --use --platform=linux/arm64,linux/amd64 --name multiplatinum
docker buildx inspect --bootstrap
sudo apt install qemu-system
sudo apt install qemu-system-aarch64
sudo apt install binfmt-support
sudo update-binfmts --enable
docker run -it --rm --privileged tonistiigi/binfmt --install all
```

I've confirmed that I can now run `aarch64` (`arm64`) Ubuntu directly with `docker run --platform linux/arm64 ubuntu uname -a`. Figuring out how best to achieve build tooling around that, well, we'll see.
